R/moduleCoverage.R

```diff
@@ -47,11 +56,12 @@ setGeneric("moduleCoverage", function(name, path) {
 #' @rdname moduleCoverage
 setMethod(
   "moduleCoverage",
-  signature(name = "character", path = "character"),
-  definition = function(name, path) {
+  signature(name = "character", path = "character", byFunctionName = "logical"),
+  definition = function(name, path, byFunctionName) {
     tmpdir <- tempdir(); on.exit(unlink(tmpdir, recursive = TRUE))
-    fnDir <- file.path(path, name, "moduleFunctions") %>%
+    fnDir <- file.path(tmpdir, name, "moduleFunctions") %>%
```

Comment:
Ok, if you are simply storing things there temporarily then use tmpdir. All transient outputs/side-effects of the function should be put in a temp location that is removed automatically.